Source/DafnyPipeline/DafnyPipeline.csproj

```diff
@@ -122,7 +122,7 @@
       <LinkBase>DafnyRuntimePython</LinkBase>
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </EmbeddedResource>
-    <EmbeddedResource Include="..\DafnyRuntime\DafnyRuntimeRust\**" LinkBase="DafnyRuntimeRust">
+    <EmbeddedResource Include="..\DafnyRuntime\DafnyRuntimeRust\**" Exclude="..\DafnyRuntime\DafnyRuntimeRust\.gitignore" LinkBase="DafnyRuntimeRust">
```

Comment:
What's this for?

Reply:
It's the second warning in the description.